src/components/EntityPage/EntityPage.test.tsx

```diff
@@ -169,4 +195,49 @@ describe('EntityPage', () => {
     await checkNotText(/git/);
     await checkForText(/custom/);
   });
+
+  it('should respect custom scorecard sort order', async () => {
+    const extensionApi: ExtensionApi = {
+      // Order "global" scorecards without any filters before scorecards with active entity filters
+      getUiExtensions(): Promise<UiExtensions> {
+        const isGlobal = (scorecard: Scorecard) => isEmpty(scorecard.excludedTags) && isEmpty(scorecard.tags) && isUndefined(scorecard.filterQuery)
+        return Promise.resolve({
+          scorecards: {
+            sortOrder: {
+              compareFn(a, b) {
+                if (isGlobal(a) && isGlobal(b)) {
+                  return a.name.localeCompare(b.name)
+                } else if (!isGlobal(a) && !isGlobal(b)) {
+                  return a.name.localeCompare(b.name)
+                } else if (isGlobal(a)) {
+                  return -1
+                }
+                return 1
+              }
+            }
+          }
+        })
+      }
+    }
+
+    const { checkForText, findByText } = renderWrapped(
+      <EntityProvider entity={entity}>
+        <EntityPage />
+      </EntityProvider>,
+      cortexApi,
+      {
+        '/': rootRouteRef,
+      },
+      [extensionApiRef, extensionApi],
+    );
+
+    // Scorecard 2 should be first because it has no entity filters
+    const scorecard1 = await findByText('Test Scorecard 1');
+    const scorecard2 = await findByText('Test Scorecard 2')
```

Comment:
I suggest explicitly running `yarn lint --fix` (to address e.g. missing EOL semicolons), and also pulling in the latest master (to get the pre-commit hook fix) to prevent future issues.
```suggestion
    const scorecard2 = await findByText('Test Scorecard 2');
```